src/tribler/core/utilities/process_manager/__init__.py

```diff
@@ -1,3 +0,0 @@
-from tribler.core.utilities.process_manager.manager import ProcessManager, get_global_process_manager, \
-    set_global_process_manager
-from tribler.core.utilities.process_manager.process import ProcessKind, TriblerProcess
```

Comment:
Observation: these unused imports were externally used in `core/components/reporter/exception_handler`, causing the tests to fail after removal.